pkgs/top-level/python-packages.nix

```diff
@@ -3740,6 +3748,10 @@ in {
 
   prompt_toolkit = callPackage ../development/python-modules/prompt_toolkit { };
 
+  # incompatible change from 1.x.x -> 2.x.x
+  # required for jupyter packages (python 2.7 LTS)
+  prompt_toolkit_1 = callPackage ../development/python-modules/prompt_toolkit/1.nix { };
```

Comment:
Maybe we should always use this prompt toolkit with 2.7. That way we prevent possible collisions. I doubt packages require a newer version, but I may be wrong. We can keep it for a separate changeset.